src/RouteConfig.js

```diff
@@ -87,8 +99,7 @@ export default (
             (argumentName: string): string => argumentName.replace(':', ''),
           );
           const values = argumentNames
-            .map((argument: string): string => request.params[argument])
-            .filter((value: ?Object): boolean => value !== undefined);
```

Comment:
Yeah, I guess we can remove this.  It was the case where you had url parameters + post body.